lib/main_common.pm

```diff
@@ -1558,6 +1560,7 @@ sub load_extra_tests_docker {
         loadtest "console/docker_image";
     }
     if (is_opensuse) {
+        loadtest "console/podman_image" if is_tumbleweed;
```

Comment:
Our tests should always consider openSUSE Tumbleweed as the default product without a version and I think this is what you normally do anyway. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check. E.g. how about `unless is_leap('15.1')` assuming that the necessary dependencies are submitted to the current version of Leap in development. Otherwise you can still use `unless is_leap`

Reply:
Currently we don't have podman in any leap version, that's why I put it there. I see no semantic difference between `if is_tumbleweed` and `unless is_leap`. If podman is meant to be in 15.1 or 15.2, it could be changed to `is_leap(">=15.x")`.

Reply:
`unless is_leap('15.1')`

Um..but then this will break on anyone running Leap 42.3 or 15.0 maintenance updates

I prefer Fabians approach for this reason